src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
"Nothing was pressed yet" is not a pressed key

Reply:
@silchyn what u mean? it works how it should due to the task. If not - please, give me more details.
![image](https://github.com/mate-academy/react_keyboard/assets/127235037/cf6fa4f9-c737-48fa-8fd5-a30437ea8ab5)